playbooks/DeploySDDCMgr.yml

```diff
@@ -0,0 +1,178 @@
+        disk_provisioning: "{{ Common.DiskProvisioning }}"
+        power_on: true
+        wait_for_ip_address: true
+        inject_ovf_env: true
+        properties:
+          ROOT_PASSWORD: "{{ Nested_SDDCMgr.Credential.root.Password }}"
+          VCF_PASSWORD: "{{ Nested_SDDCMgr.Credential.vcf.Password }}"
+          BASIC_AUTH_PASSWORD: "{{ Nested_SDDCMgr.Credential.admin.Password }}"
+          BACKUP_PASSWORD: "{{ Nested_SDDCMgr.Credential.backup.Password }}"
+          LOCAL_USER_PASSWORD: "{{ Nested_SDDCMgr.Credential.local.Password }}"
+          FIPS_ENABLE: "{{ Nested_SDDCMgr.Component.Appliance.FIPSEnable }}"
+          vami.hostname: "{{ Nested_SDDCMgr.Component.Appliance.FQDN }}"
+          guestinfo.ntp: "{{ Common.NTP.Server1.IPv4 }}"
+          vami.ip0.SDDC-Manager: "{{ Nested_SDDCMgr.Component.Appliance.Address.IPv4.Address }}"
+          vami.netmask0.SDDC-Manager:  "{{ (Nested_SDDCMgr.Component.Appliance.Address.IPv4.Address + '/' + Nested_SDDCMgr.Component.Appliance.Address.IPv4.Prefix) | ansible.utils.ipaddr('netmask') }}"
+          vami.gateway.SDDC-Manager: "{{ Nested_SDDCMgr.Component.Appliance.Address.IPv4.Gateway }}"
+          vami.domain.SDDC-Manager: "{{ Common.DNS.Domain }}"
+          vami.searchpath.SDDC-Manager: "{{ Common.DNS.Domain }}"
+          vami.DNS.SDDC-Manager: "{{ Common.DNS.Server1.IPv4 }}"
+      when:
+        - Deploy.Product.SDDCMgr.Deploy
+        - sddcmgr_check.status != 200
+
+    - name: Write annotation on the SDDCMgr VM
+      community.vmware.vmware_guest:
+        hostname: "{{ Target.FQDN }}"
+        username: "{{ Target.User }}"
+        password: "{{ Target.Password }}"
+        validate_certs: "{{ Common.PKI.ValidateCerts }}"
+        cluster: "{{ Target.Cluster }}"
+        datacenter: "{{ Target.DataCenter }}"
+        folder: "{{ Target.VMFolder }}"
+        name: "{{ Nested_SDDCMgr.Component.Appliance.VMName }}"
+        annotation: |
+                    {{ Common.Annotation }}
+                    {{ Deploy.Software.SDDCMgr.Vendor }} {{ Deploy.Software.SDDCMgr.Product }} {{ Deploy.Software.SDDCMgr.Version }}
+                    SDDCMgr root User: {{ Nested_SDDCMgr.Credential.root.Name }}
+                    SDDCMgr root Password: {{ Nested_SDDCMgr.Credential.root.Password }}
+                    SDDCMgr admin User: {{ Nested_SDDCMgr.Credential.admin.Name }}
+                    SDDCMgr admin Password: {{ Nested_SDDCMgr.Credential.admin.Password }}
+                    SDDCMgr local User: {{ Nested_SDDCMgr.Credential.local.Name }}
+                    SDDCMgr local Password: {{ Nested_SDDCMgr.Credential.local.Password }}
+                    SDDCMgr backup User: {{ Nested_SDDCMgr.Credential.backup.Name }}
+                    SDDCMgr backup Password: {{ Nested_SDDCMgr.Credential.backup.Password }}
+                    SDDCMgr vcf User: {{ Nested_SDDCMgr.Credential.vcf.Name }}
+                    SDDCMgr vcf Password: {{ Nested_SDDCMgr.Credential.vcf.Password }} 
```

Comment:
Ansible lint complains about the trailing space.

Reply:
Had to l;earn what ansible lint was... neat tool, alerts addressed.